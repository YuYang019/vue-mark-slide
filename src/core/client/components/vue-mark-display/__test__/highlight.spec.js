const marked = require('marked')
const highlight = require('../highlight')
const { getFragment } = require('./utils')

marked.setOptions({
  highlight
})

describe('highlight', () => {
    const code = getFragment('code.md')

    test('should highlight the code', () => {
        const output = marked.parse(code)
        expect(output).toMatchSnapshot()
    })
})