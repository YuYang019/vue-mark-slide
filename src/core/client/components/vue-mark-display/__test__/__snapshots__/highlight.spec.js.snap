// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`highlight should highlight the code 1`] = `
"<pre><code class=\\"language-js\\"><pre class=\\"language-javascript\\"><code>    <span class=\\"token keyword\\">const</span> text <span class=\\"token operator\\">=</span> <span class=\\"token string\\">'Hello World'</span></code></pre></code></pre>
"
`;
